code/api/src/modules/user/mutations.js

```diff
@@ -27,6 +27,23 @@ export const userSignup = {
   resolve: create
 }
 
+// Crate update
+export const userUpdate = {
+  type: UserType,
+  args: {
+    id: {
+      name: 'id',
+      type: GraphQLInt
+    },
+
+    email: {
+      email: 'email',
+      type: GraphQLString
+    }
+  },
```

Comment:
And I assume we'd add in the rest of the user info here, right

Reply:
Yep! After we add the columns to the database.